lib/dangermattic/plugins/milestone_checker.rb

```diff
@@ -77,13 +68,14 @@ def check_milestone_due_date(days_before_due: DEFAULT_DAYS_THRESHOLD, if_no_mile
 
       message_text = "This PR is assigned to the milestone [#{milestone_title}](#{milestone_url}). "
       message_text += if time_before_due_date.positive?
-                        "This milestone is due in less than #{days_before_due} days.\n"
+                        "This milestone is due in less than #{days_before_due} days.\n" \
```

Comment:
While at it, I've improved the message on the Milestones plugin, which was reported as an issue (https://github.com/Automattic/dangermattic/issues/29) by @crazytonyli.